anvio/tables/__init__.py

```diff
@@ -30,7 +30,8 @@
                          'pan': pan_db_version,
                          'genomestorage': genomes_storage_vesion,
                          'auxiliary data for coverages': auxiliary_data_version,
-                         'config': workflow_config_version}
+                         'config': workflow_config_version,
+                         'modules': kegg_modules_db_version}
```

Comment:
I think this should be called `metabolic_modules_db_version`. if we switch to MetaCyc one day, or implement something that can keep both KEGG and MetaCyc, we don't have to change an essential db name :)